atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraService.java

```diff
@@ -65,4 +72,34 @@ public static synchronized void stop() {
         daemon.deactivate();
         daemon = null;
     }
+
+    private static void waitForDaemonToStart() throws InterruptedException {
+        Thread checker = new Thread() {
+            @Override
+            public void run()
+            {
+                int iterationsToCheck = 10; // ~20s
+                InetSocketAddress endPoint;
+                try {
+                    endPoint = new InetSocketAddress(InetAddress.getLocalHost(), 9160);
+                } catch (UnknownHostException e2) {
+                    endPoint = new InetSocketAddress("localhost", 9160);
+                }
+                while (iterationsToCheck-- > 0) {
```

Comment:
You're missing out, could have been an excellent case for the goesTo operator ( https://www.reddit.com/r/shittyprogramming/comments/36ipor/til_about_the_c_goes_to_operator/ )